tests/DotenvConfigurationTest.cs

```diff
@@ -1,10 +1,10 @@
 
 /*
- * Written by Warwick Molloy GitHub|@WazzaMo
- * to ensure AspNetCore log configuration can be
- * specified as per AspNetCore documentation
- * and code templates.
- */
+* Written by Warwick Molloy GitHub|@WazzaMo
```

Comment:
Great, this comments don't belong in the code files. Not many open source projects does that and not me either.